<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestAuthentication</class>
 <widget class="QWidget" name="RequestAuthentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solicitar Autenticación</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(76, 118, 82);
background-color:rgb(216, 230, 225);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt;</string>
     </property>
     <property name="text">
      <string>Solicitar Autenticación</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>595</width>
        <height>322</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 75 12pt &quot;Noto Sans&quot;;
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Seleccione Contacto</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QListWidget" name="contactsListWidget"/>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="requestAuthentication">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 35, 21);
font: 75 9pt &quot;Noto Sans&quot;;
background-color: rgb(229, 229, 229);</string>
            </property>
            <property name="text">
             <string>Solicitar</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/enviar.png</normaloff>:/images/enviar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="addContact">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 35, 21);
font: 75 9pt &quot;Noto Sans&quot;;
background-color: rgb(229, 229, 229);</string>
            </property>
            <property name="text">
             <string>Contacto</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/agregar.png</normaloff>:/images/agregar.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="searchContact">
            <property name="placeholderText">
             <string>Buscar...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="requestAuthProgressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::BottomToTop</enum>
         </property>
         <property name="format">
          <string>Solicitando...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
